src/azul/indexer/document.py

```diff
@@ -346,7 +346,8 @@ class Document(Generic[C]):
 
     coordinates: C
     version_type: VersionType = field(init=False)
-    version: Optional[int]
+    # version is a tuple composed of the sequence number and primary term
+    version: Optional[Tuple[int, int]]
```

Comment:
Are you sure both are integers?

Reply:
I cant find documentation of the type expressed in ElasticSearch 6.x documentation, but here is a reference to the bulk API for 7.9 that states that the response `seq_num` and `primary_term` are integers. https://www.elastic.co/guide/en/elasticsearch/reference/7.9/docs-bulk.html#bulk-api-response-body

All references to the sequence_number and primary_term that I have seen in the 6.x documentation do not mention the typing for these parameters. 

unfortunately `elasticsearch-py` is also not helpful it uses `Optional[any]` 
https://github.com/elastic/elasticsearch-py/search?q=if_seq_no

If you would like me to spend more time looking for explicit documentation, I can spend more time searching.